core/api/transport/impl/http_session.cpp

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "http_session.hpp"
+
+#include <boost/config.hpp>
+#include <iostream>
+
+namespace kagome::api {
+
+  /**
+   * @brief process http request, compose and execute response
+   * @tparam Body request body type
+   * @tparam Allocator allocator type
+   * @param req request
+   * @param send sender function
+   */
+  template <class Body>
+  void HttpSession::handleRequest(boost::beast::http::request<Body> &&req) {
```

Comment:
Maybe add `using namespace boost::beast::http` or using-s for request, response, etc? Hard to read without it